db/seeds.rb

```diff
@@ -1,7 +1,107 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.fi
+cfac_2021 = Event.create!(name: "Crawfish for a Cause 2021", city: "Atlanta", state: "Georgia", location: "175 Le Brun Road", description: "The one that started it all at LeBrun")
+cfac_2022 = Event.create!(name: "Crawfish for a Cause 2022", city: "Atlanta", state: "Georgia", location: "Steady Hand Brewery", description: "The first one at Steady Hand.")
+mental_awakening = Event.create!(name: "Mental Awakening", city: "Atlanta", state: "Georgia", location: "Steady Hand Brewery", description: "Mental Awakening 2023")
+cfac_clt = Event.create!(name: "Crawfish for a Cause Charlotte", city: "Charlotte", state: "North Carolina", location: "Suffolk Punch Brewery", description: "The one in Charlotte")
```

Comment:
are these the seeds descriptions you want to display for the site or will these be modified at a later date?

Reply:
These descriptions will be modified at a later date. I am collaborating with the co-founder and one of our directors on what we want to include here.